github-marketing-strategy.md

```
@@ -0,0 +1,368 @@
+          
+      - name: Post to Slack/Discord
+        if: steps.search.outputs.results != ''
+        run: |
+          # Requires SLACK_WEBHOOK_URL in repo secrets
+          curl -X POST "${{ secrets.SLACK_WEBHOOK_URL }}" \
+            -H 'Content-Type: application/json' \
+            -d "{\"text\": \"New bounty targets:\n${{ steps.search.outputs.results }}\"}"
+```
+
+**Note:** Requires `SLACK_WEBHOOK_URL` secret configured in repo settings.
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)